lib/tls.js

```diff
@@ -189,8 +183,7 @@ exports.checkServerIdentity = function checkServerIdentity(host, cert) {
 
   if (!valid) {
     var err = new Error(
-        util.format('Hostname/IP doesn\'t match certificate\'s altnames: %j',
-                    reason));
+        `Hostname/IP doesn't match certificate's altnames: "${reason}"`);
```

Comment:
It's always a string, right?  It can't have circular references.

> As an aside, how do backticks handle circular refs?

They don't, interpolated values are simply stringified with .toString().